client/src/pages/ChatRoom/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { H1, Color } from '../../components/common';
+import React from 'react';
+import { Container } from '../../components/Container';
+import {
+  ChatInput,
+  ChatLounge,
+  MessageComponent,
+} from '../../components/chatRoom';
+import { Button } from '../../components/Button';
+import { Header } from '../../components/Header';
+
+interface Message {
+  author: string;
+  body: string;
+}
+
+const FAKE_DATA = {
```

Comment:
Essa parte aqui do fake é pra teste?

Reply:
Isso. Quando for pra pegar os dados  reais seria só substituir